app/javascript/shared/components/tags.jsx

```diff
@@ -411,9 +412,11 @@ class Tags extends Component {
     ));
     if (
       searchResults.length > 0 &&
-      (document.activeElement.id === 'tag-input' ||
-        document.activeElement.className ===
-          'articleform__tagsoptionrulesbutton')
+      (activeElement.id === 'tag-input' ||
+        activeElement.classList.contains(
+          'articleform__tagsoptionrulesbutton',
+        ) ||
```

Comment:
Just an FYI, I didn't format it like this. This is what the pre-commit linter spit out.